src/vm/hints/hint_utils.rs

```diff
@@ -16,6 +16,43 @@ use num_traits::{FromPrimitive, Signed, ToPrimitive, Zero};
 use std::collections::HashMap;
 use std::ops::{Neg, Shl, Shr};
 
+//Returns the value in the current execution scope that matches the name and is of type BigInt
+pub fn get_int_from_scope(vm: &mut VirtualMachine, name: &str) -> Option<BigInt> {
+    let mut val: Option<BigInt> = None;
+    if let Some(variables) = vm.exec_scopes.get_local_variables() {
+        if let Some(PyValueType::BigInt(py_val)) = variables.get(name) {
+            val = Some(py_val.clone());
+        }
+    }
+    val
+}
+
+//Returns the value in the current execution scope that matches the name and is of type List
+pub fn get_list_from_scope(vm: &mut VirtualMachine, name: &str) -> Option<Vec<BigInt>> {
+    let mut val: Option<Vec<BigInt>> = None;
+    if let Some(variables) = vm.exec_scopes.get_local_variables() {
+        if let Some(PyValueType::List(py_val)) = variables.get(name) {
+            val = Some(py_val.clone());
```

Comment:
I'm a bit worried about the perf of cloning a list of `BigInt`s. Would it be possible to return a slice?